lib/Basics/NumberOfCores.h

```diff
@@ -31,6 +32,14 @@ namespace NumberOfCores {
 /// @brief return number of available CPU cores
 std::size_t getValue();
 bool overridden();
-
+#ifdef USE_ENTERPRISE
+bool setValue(std::size_t);
+#endif
 }  // namespace NumberOfCores
 }  // namespace arangodb
+
+struct NumberOfCoresCache {
+  NumberOfCoresCache();
+  std::atomic<std::size_t> cachedValue;
+  std::atomic<bool> overridden;
+};
```

Comment:
This class is unimplemented. It is not the same class as `NumberOfCoresCache` defined in an anonymous namespace in `NumberOfCores.cpp`. It this declaration used anywhere?